public/locales/en/translation.json

```diff
@@ -1,17 +1,20 @@
 {
-  "welcome": {
-    "message": "Welcome to React"
+  "whoAreWe": {
+    "header": "WHO ARE WE?"
```

Comment:
i recommend doing "Who Are We"? normal casing and then using CSS text-transform to change it. this offers more flexibility with styling in the future, otherwise you need to change in 3 places